pfcli/deployment.py

```diff
@@ -355,6 +354,20 @@ def create(
     )
 
 
+@app.command()
+def scale(
```

Comment:
What do you think about let users set num_replicas when creating deployments?
Currently, two independent commands should be executed to launch multiple inferences servers in a deployment.

Reply:
Sounds good, I'll update create command.